crates/full-node/sov-ethereum/src/handlers/subscribe.rs

```
@@ -37,8 +31,11 @@ where
     S::Address: FromVmAddress<EthereumAddress>,
     Seq::Rt: HasKernel<S> + EthereumAuthenticator<S> + Default + Send + Sync + 'static,
 {
-    let eth_subscribe = parameters.parse::<EthSubscribe>()?;
-    let log_filter = match validate_params_for_log_subscription(eth_subscribe) {
+    let mut parameters = parameters.sequence();
+    let kind: SubscriptionKind = parameters.next()?;
+    let params: Params = parameters.optional_next()?.unwrap_or_default();
```

It looks like the websocket might still fail silently and get into a bad state when not enough params are provided (because of the `?`). Should we `pending.reject` with a helpful error instead of using `?`